### PR TITLE
zpython: Format head style

### PR DESCRIPTION
**Pull request by dj-m**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/zpython.rb [hunk 1]
```diff
@@ -18,10 +18,14 @@ def message
 class Zpython < Formula
   desc "Embeds a Python interpreter into zsh"
   homepage "https://bitbucket.org/ZyX_I/zsh"
+  version "00-5.0.5"
```
Comment (DomT4) [inline Formula/zpython.rb]: This should live in the `stable do` block, underneath the `url` or `mirror` where one exists.

**Comment by DomT4** [comment]
This formula is pretty much stuck in limbo at the moment, per https://github.com/Homebrew/legacy-homebrew/issues/40318, but 👍 bar one style nit on the changes here.

**Comment by dj-m** [comment]
@DomT4 Ah, thanks! I just moved it down... hopefully it gets unstuck... at some point.

**Comment by DomT4** [comment]
I suspect we'll either end up deleting the formula eventually or tweaking `zsh` to play better with it. We'll see, I guess.

Merged in https://github.com/Homebrew/homebrew-core/commit/3b372a55d0b814de9fe2f25c21c98192b35f49c4. Thank you for another contribution to Homebrew 🎉

**Comment by UniqMartin** [comment]
FYI, I made some minor adjustments in b0d2a373c78bd6e1079f7c0b5196ffb90641739d. The comment, that was supposed to be attached to the `version` line, was left behind and wasn't really making sense anymore.

**Pull request merged**